Documentation/spec/docfx_flavored_markdown.md

```diff
@@ -13,57 +16,37 @@ title: A
+For conceptual Markdown files UID can be defined by adding a `uid` metadata in YAML header:
 
-    ```md
-    ---
-    uid: uidForTheFile
-    ---
-    This is a conceptual file with `uid` specified.
-    ```
-2.  For *API Documentation*
-
-    Generally, the value of `uid` can be found in the `YAML` file generated by running `docfx metadata`. For API in DOTNET world, [2.Identifiers in Metadata Dotnet Spec](metadata_dotnet_spec.md) describes the way to generate `id` for API in dotnet language.
+```md
+---
+uid: uid_of_the_file
+---
 
-#### Syntax for cross reference
-1.  Link style: `[name](xref:uid "title")`
+This is a conceptual topic with `uid` specified.
+```
 
-    If there is a `dfm.md` file with `uid` as `dfm`:
+For reference topics, UIDs are auto generated from source code and can be found in generated YAML files.
 
-    ```md
-    ---
-    uid: dfm
-    ---
-    # DocfxFlavoredMarkdown
-    This is dfm.md
-    ```
-    We can quickly reference to this file `dfm.md` in another Markdown file using:
+You can use one of the following syntax to cross reference a topic with UID defined:
 
-    ```md
-    [DFM](xref:dfm "This is a dfm")
-    ```
-    It will transform to `<a href="dfm.md" title="This is a dfm">DFM</a>` in the final html file. If `name` empty, `docfx` uses the title of `dfm.md` as the default name, for example, `[](xref:dfm)` will be transformed to `<a href="dfm.md">DocfxFlavoredMarkdown</a>`.
+1. Markdown link: `[link_text](xref:uid_of_the_topic)`
+2. Auto link: `<xref:uid_of_the_topic>`
+3. Shorthand form: `@uid_of_the_topic`
 
-2.  Autolink style: `<xref:uid>`
+All will render to:
 
-    Still take `dfm.md` as an example, `<xref:dfm>` will transform to `<a href="dfm.md">DocfxFlavoredMarkdown</a>`. The title of `dfm.md` is taken as the display name.
+```html
+<a href="url_of_the_topic">link_text</a>
+```
 
-3.  Shortcut: `@uid`
+If `link_text` is not specified, DocFX will extract the title from the target topic and use it as the link text.
 
-    Syntax for `@uid` is as below:
-    1. If content after `@` is wrapped by `'` or `"`,  it contains any character including white space.
-    2. If content after `@` is not wrapped by `'` or `"`, it must start with word character `a-z` or `A-Z`, and it ends when:
-        1. line ends
-        2. meets whitespaces
-        3. line ends with `.`, `,`, `;`, `:`, `!`, `?` and `~`
-        4. meets 2 times or more `.`, `,`, `;`, `:`, `!`, `?` and `~`
+For more information, see [cross reference](../tutorial/links_and_cross_references.md#using-cross-reference).
 
-    So `@dfm` will also be transformed to `<a href="dfm.md">DocfxFlavoredMarkdown</a>`. The title of `dfm.md` is taken as the display name.
 
 ### File Inclusion
 DFM adds syntax to include other file parts into current file, the included file will also be considered as in DFM syntax. *NOTE* that YAML header is **NOT** supported when the file is an inclusion.
```

Comment:
line 104: if none of the first three option is specified. -> if none of the first three options are specified.